kymatio/scattering2d/frontend/torch_frontend.py

```diff
@@ -0,0 +1,110 @@
+__all__ = ['Scattering2DTorch']
+
+import torch
+import torch.nn as nn
+
+from .base_frontend import Scattering2DBase
+from kymatio.scattering2d.core.scattering2d import scattering2d
+from ...frontend.torch_frontend import ScatteringTorch
+
+
+class Scattering2DTorch(ScatteringTorch, Scattering2DBase):
+    def __init__(self, J, shape, L=8, max_order=2, pre_pad=False,
+    backend='torch'):
+        ScatteringTorch.__init__(self)
+        Scattering2DBase.__init__(self, J, shape, L, max_order, pre_pad, backend)
+        Scattering2DBase._instantiate_backend(self,
+                'kymatio.scattering2d.backend.')
+        Scattering2DBase.build(self)
+        Scattering2DBase.create_filters(self)
+
+        self.register_filters()
+
+
+    def backend_setup(self):
+        if not self.backend:
+            from ..backend.torch_backend import backend
+            self.backend = backend
+        elif not self.backend.name.startswith('torch'):
+            raise RuntimeError('This backend is not supported.')
```

Comment:
Remove.

Reply:
Why? So I thought the new mechanism would have to handle backened names. I'm going to add it.